tests/minimax.wcon

```diff
@@ -27,7 +27,10 @@
 		"ox":"mm",
 		"speed":"mm/s",
 		"curvature":"1/mm",
-		"width":"mm"
+		"width":"mm",
+		"humidity":"%",
+		"temperature":"C",
+        "age":"h"
```

Comment:
I turns out indentation was inconsistent in `minimax.wcon`, the schema, and in a couple of the .MD examples, and I've fixed this so all indentation is 4 spaces (no tabs)